.circleci/config.yml

```diff
@@ -278,19 +277,7 @@ jobs:
                exit 1
             fi
 
-      - run:
```

Comment:
Should this be just commented out for now so that we don't have to rewrite this part when we are ready to publish again? Or do you feel it is risky to hold on to this code at the moment?

Reply:
Eh. We can always look at git history if we need to see it again.

Reply:
@rmohan20 - upgrading to v4 of the orb will require this to be rewritten. It seemed safest to remove it while @jfeingold35 reworks the overall flow of the CI/CD workflows.

Reply:
Makes sense 👍